pkg/virt-launcher/virtwrap/api/converter_test.go

```diff
@@ -1330,6 +1330,40 @@ var _ = Describe("Converter", func() {
 	})
 })
 
+var _ = Describe("getSRIOVPciAddresses", func() {
```

Comment:
I miss a full conversion from VMI to Domain which uses some SRIOV devices. Also, I would prefer if we put the SRIOV-VF-PCI-ADDR into the converter context and look it up from outside the converter.

Reply:
I think @rmohr suggested to store SRIOV-VF-PCI-ADDR in the ConverterContext [1] instead of using a env var.

[1] https://github.com/kubevirt/kubevirt/blob/900c672c57cb2b3cdf7d516b90110d64bc45b17c/pkg/virt-launcher/virtwrap/api/converter.go#L54

Reply:
@vladikr thanks!

The suggestion makes sense in scope of the current DP implementation where we deal with a single variable. My only concern is that when we eventually move to multiple variables as was just merged to sriovdp master today, we may want to extract information for specific networks that are actually attached to the VMI which implies a bit more knowledge about the VMI spec than perhaps is fitting for the caller.

Anyway, we can revisit it later.